### DISCUSSION TITLE
🔥🔥🔥Could you please tell me what is your favourite month?

### DISCUSSION THREAD

**Discussion by shanpeng-m** [NAIST-SE/AttractingContributors#804] [2023 Attraction Contributions]
Could you please tell me what is your favourite month?

https://github.com/inusturbo/AttractingContributors-month

# How to? 💻

Step1. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163042-sLcxKT.png" alt="image-20230612163041929" style="zoom:33%;" />

Step2. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163122-0KWZKV.png" alt="image-20230612163122150" style="zoom: 25%;" />

Step3. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163509-QEGXRB.png" alt="image-20230612163508909" style="zoom: 33%;" />

Step4. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163614-g4VTg9.png" alt="image-20230612163614305" style="zoom:50%;" />

Step5. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163658-AfDpc0.png" alt="image-20230612163658536" style="zoom:50%;" />

Step6. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163727-5sC6Su.png" alt="image-20230612163727003" style="zoom:33%;" />

Step7. 

<img src="https://cdn.jsdelivr.net/gh/inusturbo/images@main/uPic/20230612-163745-ouvkqZ.png" alt="image-20230612163745286" style="zoom: 33%;" />

Done! Good job! I will merge it. Thank you in advance for your contribution, and I will help you to do it too.

**Please left your Repo URL in this discussion, I will help you too!**

**Comment by KKKKKXY** [comment]
I added my favorite month into your list.(githubid: KKKKKXY)

**Comment by Sanaa3131** [comment]
Hi, I couldn't merge your pull request because of a branch conflict, do you know how to solve it? BTW, I have already added my favorite month in your list.
